challenge.rb

```diff
@@ -67,6 +96,33 @@ def ac_monitor current_temp, ac_working, desired_temp
 # and print out the result of `fizz_buzz_calculator` for that number.
 # This question will be graded independently of question four.
 
-# Replace this comment with your code.
+def fizzbuzz_calc (min max)
+  list = {
+  'Fizz' => [],
+  'Buzz' => [],
+  'FizzBuzz' =>[],
+  'other' => []}
+
+  i = 1
+  while i <= max && i > max
```

Comment:
This statement will always be true; consequently, we will never leave the `while` loop.